lib/rex/post/meterpreter/ui/console/interactive_channel.rb

```diff
@@ -75,7 +75,7 @@ def _interact_complete
   #
   def _stream_read_local_write_remote(channel)
     if raw
-      data = user_input.getch
+      data = user_input.sysread(1024)
     else
       data = user_input.gets
     end
```

Comment:
Unrelated to this PR, but also noticing that there's a call to handle a line of user input:
https://github.com/rapid7/metasploit-framework/blob/95741007607eedbe46a45e302059cf3379f5fd0d/lib/rex/post/meterpreter/ui/console/interactive_channel.rb#L84

Which interactive shells may break the assumptions of